components/scream/src/control/atmosphere_driver.cpp

```diff
@@ -112,6 +141,129 @@ void AtmosphereDriver::initialize (const ekat::Comm& atm_comm,
   register_groups();
   m_field_repo->registration_ends(m_grids_manager);
 
+  m_ad_status |= s_fields_created;
+}
+
+void AtmosphereDriver::initialize_output_manager () {
+  check_ad_status (s_comm_set | s_params_set | s_grids_created | s_fields_created);
+
+  // Create Initial conditions
+
+  // Create Output manager
+  if (m_atm_params.isSublist("Output Manager")) {
+    auto& out_params = m_atm_params.sublist("Output Manager");
+    m_output_manager.set_params(out_params);
+    m_output_manager.set_comm(m_atm_comm);
+    m_output_manager.set_grids(m_grids_manager);
+    m_output_manager.set_repo(m_field_repo);
+  }
+  m_output_manager.init();
+
+  m_ad_status |= s_output_inited;
+}
+
+void AtmosphereDriver::
+initialize_fields (const util::TimeStamp& t0)
```

Comment:
entirely unrelated, so take more as a comment that came to mind while reading this.  Is it possible to have every DAG output (.dot) file have a boiler plate comment in it that explains how to visualize it?  I know that when there are certain error messages that provide this info, but in the case explained in the comment below we won't get that error message, which then means the user has the DAG but has to be proficient in how to work with the `.dot` file to actually get any use out of it.